<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainForm</class>
 <widget class="QWidget" name="MainForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1240</width>
    <height>927</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>主页面</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../resource/resource.qrc">
    <normaloff>:/icon/main.ico</normaloff>:/icon/main.ico</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QGridLayout" name="mainLayout">
     <item row="22" column="0">
      <layout class="QGridLayout" name="typeCountLayout" columnstretch="0,0">
       <property name="verticalSpacing">
        <number>12</number>
       </property>
       <item row="6" column="0">
        <layout class="QGridLayout" name="typeCountView">
         <item row="1" column="0">
          <widget class="QChartView" name="allTypeVieaw"/>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="label_14">
           <property name="text">
            <string>消费统计</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="6" column="1">
        <layout class="QGridLayout" name="typeView">
         <item row="0" column="0">
          <widget class="QLabel" name="label_15">
           <property name="text">
            <string>本月消费统计</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QChartView" name="monthTypeView"/>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item row="4" column="0">
      <layout class="QGridLayout" name="recordLayout">
       <item row="0" column="1">
        <layout class="QGridLayout" name="recordListlayout">
         <item row="3" column="2">
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>收支类型</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>编号</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item row="3" column="3">
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>财富类型</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item row="2" column="4">
          <widget class="QPushButton" name="addHistory">
           <property name="text">
            <string>添加记录</string>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QLabel" name="label_6">
           <property name="text">
            <string>财富(元)</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item row="3" column="4">
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>消费时间</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item row="4" column="0" colspan="5">
          <widget class="QListView" name="historyListView">
           <property name="minimumSize">
            <size>
             <width>350</width>
             <height>0</height>
            </size>
           </property>
           <property name="textElideMode">
            <enum>Qt::ElideMiddle</enum>
           </property>
           <property name="resizeMode">
            <enum>QListView::Adjust</enum>
           </property>
           <property name="uniformItemSizes">
            <bool>true</bool>
           </property>
           <property name="batchSize">
            <number>125</number>
           </property>
           <property name="wordWrap">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_13">
           <property name="text">
            <string>本月记录</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="0" column="0">
        <layout class="QGridLayout" name="coutView">
         <item row="1" column="1">
          <widget class="QLabel" name="totalMoneyLabel">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>50</height>
            </size>
           </property>
           <property name="text">
            <string>总财富</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
           <property name="margin">
            <number>5</number>
           </property>
          </widget>
         </item>
         <item row="3" column="2">
          <widget class="QLabel" name="todayCountLabel">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>50</height>
            </size>
           </property>
           <property name="text">
            <string>今日记录数</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
           <property name="margin">
            <number>5</number>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QLabel" name="todayMoneyLabel">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>50</height>
            </size>
           </property>
           <property name="text">
            <string>今日财富</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
           <property name="margin">
            <number>5</number>
           </property>
          </widget>
         </item>
         <item row="0" column="1" colspan="2">
          <widget class="QLabel" name="label">
           <property name="text">
            <string>统计预览</string>
           </property>
          </widget>
         </item>
         <item row="4" column="2">
          <widget class="QLabel" name="todayCount">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>50</height>
            </size>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
           <property name="margin">
            <number>8</number>
           </property>
          </widget>
         </item>
         <item row="2" column="2">
          <widget class="QLabel" name="totalCount">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>50</height>
            </size>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
           <property name="margin">
            <number>8</number>
           </property>
          </widget>
         </item>
         <item row="1" column="2">
          <widget class="QLabel" name="totalCountLabel">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>50</height>
            </size>
           </property>
           <property name="text">
            <string>总记录数</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
           <property name="margin">
            <number>5</number>
           </property>
          </widget>
         </item>
         <item row="4" column="1">
          <widget class="QLabel" name="todayMoney">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>50</height>
            </size>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
           <property name="margin">
            <number>8</number>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QLabel" name="totalMoney">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>50</height>
            </size>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
           <property name="margin">
            <number>8</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item row="6" column="0">
      <layout class="QGridLayout" name="countLayout">
       <item row="1" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <layout class="QGridLayout" name="consumeLayout">
           <item row="1" column="0">
            <widget class="QChartView" name="consumeView"/>
           </item>
           <item row="0" column="0">
            <widget class="QLabel" name="label_7">
             <property name="text">
              <string>消费走势</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QGridLayout" name="anyView">
           <item row="0" column="0">
            <widget class="QLabel" name="label_12">
             <property name="text">
              <string>收入走势</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QChartView" name="anyGraphicsView"/>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QChartView</class>
   <extends>QGraphicsView</extends>
   <header location="global">qchartview.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../resource/resource.qrc"/>
 </resources>
 <connections/>
</ui>
